src/Settings/ParenthesesPage.cpp

```diff
@@ -44,15 +44,15 @@ ParenthesisWidget::ParenthesisWidget(const QString &language, QChar leftParenthe
 
     mainLayout->addSpacing(20);
 
-    stretchLayout = new QHBoxLayout();
+    auto stretchLayout = new QHBoxLayout();
     mainLayout->addLayout(stretchLayout);
 
     stretchLayout->addStretch();
 
-    checkBoxesLayout = new QVBoxLayout();
+    auto checkBoxesLayout = new QVBoxLayout();
     stretchLayout->addLayout(checkBoxesLayout);
 
-    auto addOption = [this](QCheckBox *&checkBox, const QString &name, Qt::CheckState state) {
+    auto addOption = [this, &checkBoxesLayout](QCheckBox *&checkBox, const QString &name, Qt::CheckState state) {
```

Comment:
```suggestion
    auto addOption = [this, checkBoxesLayout](QCheckBox *&checkBox, const QString &name, Qt::CheckState state) {
```

It's better not to pass a local variable by reference in a lambda. It's safe here since it's not a callback function, i.e. it won't be called outside of `ParenthesisWidget::ParenthesisWidget` where `checkBoxesLayout` (the pointer itself, not what it points to) is destroyed. However, it doesn't cost more to pass by value as `checkBoxesLayout` is a pointer, so it's better to follow the convention.